events/interaction/handlers/button.js

```diff
@@ -1,8 +1,18 @@
 import { handleCreateUserButton } from "../features/users/create-user-modal.js";
+import { handleClockInButton } from "../features/sessions/handle-clock-in-button.js";
+import { handleClockOutButton } from "../features/sessions/handle-clock-out-button.js";
 
 export const buttonHandlers = {
   create_user_modal_button: {
     run: handleCreateUserButton,
+    context: { needsUser: false, needsSession: false },
```

Comment:
This was true as a null response from the db tells me the user doesn't already exist or if there is one it tells me they do.
```suggestion
    context: { needsUser: true, needsSession: false },
```